cli/lib/kontena/client.rb

```diff
@@ -321,14 +327,15 @@ def request(http_method: :get, path:'/', body: nil, query: {}, headers: {}, resp
       }.merge(host_options)
 
       request_options.merge!(response_block: response_block) if response_block
+      @last_request = request_options
```

Comment:
Getting rid of @last_response would require that `client.get(...)` etc would return a response object with `.success?`, `.data[]` etc, which would require changing everything that uses the client anywhere.